src/prefect/cli/run.py

```diff
@@ -438,6 +438,12 @@ def cast_value(value: str) -> Any:
     ),
     is_flag=True,
 )
+@click.option(
+    "--schedule",
+    "-s",
+    help="Execute the flow run according to the schedule defined with the flow.",
```

Comment:
```suggestion
    help="Execute the flow run according to the schedule attached to the flow.",
```

And perhaps something like..

> When this flag is set, this command will wait between scheduled flow runs.
> If the flow has no schedule, this flag will be ignored.
> If used with a non-local run, an exception will be thrown.

Reply:
We should also throw an exception if someone passes this flag for a non-local run.